src/style.css

```diff
@@ -1 +1,62 @@
-/* add styles here */
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Book.ttf);
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-weight: 900;
+}
+
+.form__input {
+  box-sizing: border-box;
+  display: flex;
+  width: 100%;
+  padding: 0;
+  margin: 20px 0;
+
+  font-family: Avenir, Arial, Helvetica, sans-serif;
```

Comment:
Great job on using a fallback font family! This ensures that if the Avenir font doesn't load for some reason, the browser will use Arial, Helvetica, or a generic sans-serif font instead.